its/pom.xml

```diff
@@ -9,17 +9,33 @@
   </parent>
 
   <artifactId>python-its</artifactId>
-  <name>Python :: IT</name>
   <packaging>pom</packaging>
 
+  <name>Python :: ITs</name>
+
   <modules>
     <module>plugin</module>
     <module>ruling</module>
   </modules>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <skipTests>${skip.its}</skipTests>
```

Comment:
We should add `-Dskip.its=false` in `cix.sh`
And a nice to have: `cd its` , `mvn clean install -Pits` in `README.md`